RecoJets/JetPlusTracks/python/jetPlusTrackAddonSeedProducer_cfi.py

```diff
@@ -0,0 +1,13 @@
+import FWCore.ParameterSet.Config as cms
+
+jetPlusTrackAddonSeedProducer = cms.EDProducer(
```

Comment:
this file should be removed, an equivalent is already created by the framework via the ```JetPlusTrackAddonSeedProducer::fillDescriptions``` call